server.go

```diff
@@ -17,6 +17,7 @@ type Server struct {
 	MaxConns                   int64
 	MaxConcurrentTLSHandshakes int64
 	BufferSize                 int
+	Timeout                    time.Duration // Service timeout. Default is 500ms
```

Comment:
"Default is 500ms" does not match "Timeout:    3 * time.Second,". Maybe we can remove whole comment.